packages/manager/modules/exchange/src/account/update/account-update.html

```diff
@@ -487,6 +497,24 @@
                             </div>
                         </div>
                     </div>
+
+                    <!--ExchangeGuid-->
+                    <div class="row">
+                        <div class="col-md-6 oui-field mb-4">
+                            <label
+                                class="oui-field__label oui-label"
+                                for="accountExchangeGuid"
```

Comment:
maybe not required? is there a field with this ID?

Reply:
No it is not necessary because it is not a field but a data return.
I removed it.